index.html

```diff
@@ -5,6 +5,9 @@
     <title>Jump Game</title>
```

Comment:
How about changing the name to "Dinosaur Game"?

Reply:
will do
sure